lib/google-music-electron.js

```diff
@@ -216,6 +226,13 @@ function launchGme() {
     gme.browserWindow.on('minimize', gme.toggleVisibility);
   }
 
+  // When we recieve the close event, before we clean up, save the current size and position of the window
+  // DEV: Note this fires when we still have access to the browserWindow, before 'closed'
+  gme.browserWindow.on('close', function handleCloseEvent () {
+    logger.debug('Setting window size in config.');
+    config.set('window-info', gme.browserWindow.getBounds());
+  });
```

Comment:
Tried this out but am concerned that we don't save settings when a crash occurs (I would expect same location even if `electron` hard crashes). Can we switch to the events `resize` and `move`?

https://github.com/electron/electron/blob/v1.2.1/docs/api/browser-window.md#event-resize